scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/RecurrenceGoal.java

```diff
@@ -98,44 +98,44 @@ protected RecurrenceGoal fill(RecurrenceGoal goal) {
    * exist over a timespan longer than the allowed range (and one should
    * probably be created!)
    */
-  public java.util.Collection<Conflict> getConflicts(Plan plan) {
+  @Override
+  public java.util.Collection<Conflict> getConflicts(final Plan plan) {
     final var conflicts = new java.util.LinkedList<Conflict>();
 
     //collect all matching target acts ordered by start time
     //REVIEW: could collapse with prior template start time query too?
-    final var satisfyingActSearch = new ActivityExpression.Builder()
+    final var matchingActivities = new java.util.LinkedList<>(plan.find(new ActivityExpression.Builder()
```

Comment:
Have you encountered issues with this goal to start a refactoring? In PR #113, its behavior is also refactored.

Reply:
I dropped that commit, so this PR no longer touches `RecurrenceGoal.java` :+1: